students/WWhite/lesson05/Mailroom.py

```diff
@@ -102,16 +113,15 @@ def send_letters():
 if __name__ == "__main__":  # If this is the main file, run the below
 
     while True:
-        str_input = menu_options()  # Run the menu options function and save the input
-        if str_input == '1':  # If user entered 1, get the name and run the add_donation function
+        option_selected = menu_options()  # Run the menu options function and save the input
+        if option_selected == 1:  # If user entered 1, get the name and run the add_donation function
             name = switch_func_dict.get(1)()
             switch_func_dict.get(2)(name)
 
-
-        elif str_input == '2':  # If the user enters 2, create a report and then return to the original report
+        elif option_selected == 2:  # If the user enters 2, create a report and then return to the original report
             switch_func_dict.get(3)()
 
-        elif str_input == '3':  # If the user enters 3, create letters for everyone, saved in individual text files
+        elif option_selected == 3:  # If the user enters 3, create letters for everyone, saved in individual text files
```

Comment:
this block should be simplified a bit, the idea of using a "switch dict" is to actually seemlessly have your program switch to various flows, for example:
```
switch_dict = {"1": func1, "2": func2}
user_in = input("")
try:
    switch_dict[user_in]()
except KeyError:
    print("invalid option")
```